[33m[2025-02-14  21:18:10.772] _internal.py -> _log line:97 [WARNING] :  * Debugger is active![0m
[32m[2025-02-14  21:18:10.778] _internal.py -> _log line:97 [INFO] :  * Debugger PIN: 141-439-431[0m
[32m[2025-02-14  21:18:27.342] _internal.py -> _log line:97 [INFO] :  * Detected change in 'c:\\Users\\xiaoxuan\\OneDrive\\桌面\\programming\\python\\blog\\routes\\auth.py', reloading[0m
